labels.yaml

```diff
@@ -2,4 +2,473 @@ orgs:
+    color: "A8D695"
+  - name: "Awaiting Action/Author"
+    color: "9400D3"
+    aliases:
+      - "awaiting action::author"
+  - name: "backend"
+    color: "C39953"
+  - name: "breaking change"
+    color: "9400D3"
+  - name: "candidate"
+    color: "C39953"
+  - name: "CI/CD"
+    color: "E6E6FA"
+    aliases:
+      - "ci/cd"
+  - name: "cli"
+    color: "D9534F"
+  - name: "collaboration"
+    color: "AD4363"
+  - name: "community"
+    color: "8E44AD"
+  - name: "Community-Contributed PR"
+    color: "E6E6FA"
+    aliases:
+      - "community-contributed pr"
+      - "community-contributed mr"
+  - name: "configuration"
+    color: "7F8C8D"
+  - name: "customer prospect"
+    color: "004E00"
+  - name: "customer success"
+    color: "004E00"
+  - name: "customer support"
+    color: "7F8C8D"
+  - name: "d&i"
+    color: "CC338B"
+  - name: "dashboard"
+    color: "5CB85C"
+  - name: "data science / ml"
+    color: "FFECDB"
+  - name: "demo day::shared"
+    color: "C39953"
+  - name: "demo day::to share"
+    color: "ED9121"
+  - name: "demo day::up next"
+    color: "F7E7CE"
+  - name: "deployment"
+    color: "D1D100"
+  - name: "deps"
+    color: "5CB85C"
+  - name: "discussion"
+    color: "AD8D43"
+  - name: "docker"
+    color: "428BCA"
+  - name: "dogfooding"
+    color: "8E44AD"
+  - name: "enterprise edition ideas"
+    color: "69D100"
+  - name: "exploration"
+    color: "F0AD4E"
+  - name: "extract"
+    color: "0033CC"
+  - name: "frontend"
+    color: "ED9121"
+  - name: "hack day"
+    color: "E6E6FA"
+  - name: "handbook"
+    color: "3CB371"
+  - name: "hired"
+    color: "009966"
+  - name: "housekeeping"
+    color: "D10069"
+  - name: "incident"
+    color: "CC0033"
+  - name: "installation"
+    color: "FF0000"
+  - name: "integrations"
+    color: "6699CC"
+  - name: "internal"
+    color: "CC0033"
+  - name: "legal"
+    color: "5843AD"
+  - name: "load"
+    color: "0033CC"
+  - name: "making money"
+    color: "5CB85C"
+  - name: "meltano - retrospective"
+    color: "0033CC"
+  - name: "meltano culture and values"
+    color: "9400D3"
+  - name: "meltano.com"
+    color: "5843AD"
+  - name: "model"
+    color: "0033CC"
+  - name: "needs refinement"
+    color: "C21E56"
+  - name: "needs research spike"
+    color: "C21E56"
+  - name: "notebook"
+    color: "0033CC"
+  - name: "offboarding"
+    color: "C21E56"
+  - name: "onboarding"
+    color: "00B140"
+  - name: "office hours::discussed"
+    color: "36454F"
+  - name: "office hours::to discuss"
+    color: "6699CC"
+  - name: "office hours::up next"
+    color: "0000FF"
+  - name: "orchestrate"
+    color: "0033CC"
+  - name: "performance"
+    color: "D9534F"
+  - name: "plugin management"
+    color: "6699CC"
+  - name: "process"
+    color: "808080"
+  - name: "product"
+    color: "428BCA"
+  - name: "product vision"
+    color: "0033CC"
+  - name: "project awesome"
+    color: "6699CC"
+  - name: "rca"
+    color: "CC0033"
```

Comment:
It's not clear what "rca" means, and there is no description to clarify.

Reply:
root cause analysis